tools/istio-iptables/pkg/builder/iptables_builder.go

```diff
@@ -42,4 +42,13 @@ type IptablesConsumer interface {
 type IptablesBuilder interface {
 	IptablesProducer
 	IptablesConsumer
+	//TODO(abhide): Remove New*() and Flush*() methods if possible
+	// NewChainV4 creates new IPv4 chain
+	NewChainV4(chain string, table string) IptablesBuilder
+	// NewChainV6 creates new IPv6 chain
+	NewChainV6(chain string, table string) IptablesBuilder
+	// FlushChainV4 flushes Ipv4 chain
+	FlushChainV4(chain string, table string) IptablesBuilder
```

Comment:
Those new `FlushChain*` methods are not used anywhere. Remove them in this PR.
Only the chain cleanup functions may flush chains. But those functions are used only for testing, and we don't need to use the builder pattern and `iptables-restore`, etc. for those.

Reply:
FlushChainV6 is used: https://github.com/istio/istio/pull/17914/files#diff-7737d52cc5c9db7a81451826f15ec97eR286

Equivalent to the bash script: https://github.com/istio/istio/blob/master/tools/packaging/common/istio-iptables.sh#L572